doc/stdenv/stdenv.chapter.md

```diff
@@ -265,8 +265,14 @@ Dependencies can be broken down along three axes: their host and target platform
 
 The extension of `PATH` with dependencies, alluded to above, proceeds according to the relative platforms alone. The process is carried out only for dependencies whose host platform matches the new derivation’s build platform i.e. dependencies which run on the platform where the new derivation will be built. [^footnote-stdenv-native-dependencies-in-path] For each dependency \<dep\> of those dependencies, `dep/bin`, if present, is added to the `PATH` environment variable.
 
-A dependency is said to be **propagated** when some of its other-transitive (non-immediate) downstream dependencies also need it as an immediate dependency.
-[^footnote-stdenv-propagated-dependencies]
+A dependency is said to be **propagated** when some of its other-transitive
+(non-immediate) downstream dependencies also need it as an immediate dependency;
+for example, `dep:2` is propagated to `dep:0` (through `dep:1`) in the following
+diagram, as `dep:2` is a dependency of `dep:0` both immediately and transitively
+(through `dep:1`):
+
+![dep:2 is propagated to dep:0, as it is a dependency of dep:0 both
+immediately and transitively (through dep:1).](./propagation.svg)
```

Comment:
@infinisil, I couldn't find recommendations about figures in [doc/README.md](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md)

Reply:
It only documents syntax extensions not in standard CommonMark, for anything else it links to [CommonMark](https://commonmark.org/), which does document images.

Reply:
I was thinking about figures as images + captions, which, indeed don't seem to be part of CommonMark and would have to be described in our guidelines.

Maybe it was part of the docbook migration guide, because docbook did have figures, but I might have just made that up.